committee-steering/sig-governance.md

```diff
@@ -0,0 +1,629 @@
+
+#### Selecting project leads
+
+SIG ~~Foo~~ maintains 2-3 project leads.  When a project lead steps
+down from their position, they may propose a new project lead (must be
+a SIG ~~Foo~~ member) to take their place.  Lazy consensus by all SIG
+members should follow.  If [lazy consensus][lazy-consensus] cannot be
+achieved and there is no alternative clearly preferred by the majority
+of SIG, then by default the proposal is upheld and the motion passes.
+In rare cases, this decision may be escalated to the Steering Committee.
+
+If deemed necessary by the members of SIG ~~Foo~~, the number of
+project leads may be expanded.  This may be proposed by SIG members
+and determined by lazy consensus or fallback on a
+[2/3 super majority][super-majority].
+
+Project leads that have not been active for a period of 8 weeks or more
+(exceptions exist) should step down and propose a replacement.  For
+planned absences, project leads should find a temporary replacement
+to fill their role and delegate responsibility.
+
+When possible, it is preferred that the project lead group represent a
+diverse set of organizations.  This is predicated on interest in
+the role from multiple organizations with qualified candidates.
+
+### Technical Lead (TL) role
+
+Canonical list of tech leads: kubernetes-sig-~~foo~~-techleads@googlegroups.com
+This list may be used to escalate issues to technical leads.
+
+Technical leads:
+
+- Provide technical guidance to members and resolve technical disputes
+- Provide technical direction and set priorities
+- Should be well versed in most technical areas governed by the SIG
+- Should have shown consistent excellent technical judgement and
+  leadership over a sustained period
+- May appear in OWNERs files as approvers, but not
+  every member who is an approver is a technical lead for the SIG.
+- May also be project leads, however the membership
+  and responsibilities of each role is tracked separately.
+- May retain the title "~~Foo~~ technical lead emeritus" when stepping
+  down.
+
+#### Responsibilities towards members the SIG
+
+- Acting as role model within the SIG
+  - Other members will look up to technical leads in the SIG.  Leads
+    must model respectful behavior towards all members of the community
+- Acting as an escalation point for technical disputes within the SIG
+- Ensuring technical excellence within area
+  - Actively working to reduce complexity of the area
+  - Defining general acceptance criteria for work
+    - Code style guide
+    - Go doc style guide
+    - Test requirements
+    - Documentation requirements
+  - Making technical decisions with other technical leads
+    - For technical decisions the impact other areas of the product,
+      work cross organizationally to build consensus between relevant
+      parties.
+  - Making technical decisions unilaterally if critical and urgent need
+    arises
+  - Helping review and approve PRs + resolve discussion on PRs
```

Comment:
This is a role of approvers and owners. If the lead were an owner, which I think they should be, they'd retain that responsibility.